skrf/media/physicalLine.py

```diff
@@ -0,0 +1,177 @@
+
+
+'''
+.. module:: skrf.media.physicalLine
+
+========================================
+physicalLine (:mod:`skrf.media.physicalLine`)
+========================================
+
+A generic physical transmission line media.
+
+Represents a generic physical transmission line, defined by impedance,
+attenuation, relative permittivity and loss angle.
+
+Frequency invariant parameters or Djirdjevic/Svennson dispersion is provided. 
+
+
+
+'''
+from scipy.constants import  epsilon_0, c
+from numpy import real, imag, sqrt, ones, zeros, pi, log
+from .media import Media
+
+class PhysicalLine(Media):
+    '''
+    A generic physical transmission line media.
```

Comment:
this looks like the old docstring, and doesnt explain your new parameters

Reply:
Oops. Will fix that this weekend.

Reply:
Docstring updated in 9bf9ca9